### DISCUSSION TITLE
Geometry not modified

### DISCUSSION THREAD

**Discussion by aliasatp** [opengisch/QField#3919] [Bugs]
Hi,
in the last few days I realised that in section "changes" of my qfieldcloud dashboard I have "geometry not modified" message even if I create new geometry. In some cases a strange behaviour occurs, I mean the "upload" of a double copy of the same feature (2 points instead of the one created!)
Thanks for any news
Giovanni

**Comment by suricactus** [comment]
When you create a new feature you see the "geometry not modified" issue due to UX glitch we are aware of and will fix soon. The second issue you described needs a separate detailed bug report with steps to reproduce, username and project name so we can investigate.

**Comment by stempest123** [comment]
Had the "geometry not modified" issue  been resolved yet? I am experiencing this issue today
![image](https://user-images.githubusercontent.com/128692753/227166307-c6e9040a-845c-4b5e-a9d6-dbf09c66f82d.png)

**Comment by Sarika2023** [comment]
i'm getting the same error from feb 2023, problem still persists. didn't resolved yet i guess.
